### DISCUSSION TITLE
38778909-864A-0AB5-BC56-DE2C17F189DB

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#38] [General]
# 38778909-864A-0AB5-BC56-DE2C17F189DB

The Orion III Spaceplane is an orbital spacecraft owned by Pan American World Airways and used in transit from Earth to Space Station V. This is the vehicle used by Dr. Heywood Floyd upon arriving at the station during the 

https://celestia.mobi/addon?item=38778909-864A-0AB5-BC56-DE2C17F189DB&lang=en

**Comment by Dirgham2015** [comment]
Uh
